app/javascript/pages/RegisterNewUser.jsx

```diff
@@ -1,105 +1,81 @@
-import React, { useCallback, useState } from "react";
-import { useNavigate } from 'react-router-dom';
+import React, {useCallback, useState} from "react";
+import {useNavigate} from 'react-router-dom';
 
 import makeApiRequest from "../api/makeApiRequest";
 
 const RegisterNewUser = () => {
-  const [ createUserPayload, setCreateUserPayload ] = useState({});
-  const navigate = useNavigate();
+    const [createUserPayload, setCreateUserPayload] = useState({});
+    const navigate = useNavigate();
 
-  const handlePayloadUpdate = useCallback((key, value) => {
-    const updatedPayload = {
-      ...createUserPayload,
-      [key]: value
-    }
+    const handlePayloadUpdate = useCallback((key, value) => {
+        const updatedPayload = {
+            ...createUserPayload,
+            [key]: value
+        }
 
-    setCreateUserPayload(updatedPayload);
-  }, [createUserPayload, setCreateUserPayload]);
+        setCreateUserPayload(updatedPayload);
+    }, [createUserPayload, setCreateUserPayload]);
 
-  const onSubmit = useCallback(async () => {
-    const { firstName, lastName, username, email, password } = createUserPayload;
-    const finalPayload = {
-      name: `${firstName?.trim()} ${lastName?.trim()}`,
-      username: username?.trim(),
-      email: email?.trim(),
-      password: password
-    };
+    const onSubmit = useCallback(async () => {
+        const {username, email, password} = createUserPayload;
+        const finalPayload = {
+            username: username?.trim(),
+            email: email?.trim(),
+            password: password
+        };
 
-    try {
-      const _list = await makeApiRequest("/api/users", {
-        method: 'POST',
-        body: finalPayload,
-      });
-      
-      navigate("/signIn");
-    } catch (err) {
-      console.error("ERROR!", err);
```

Comment:
I think we should keep this catch block.  I realize it doesn't do anything useful, but try without catch is weird.

I plan on introducing better error handling in the next couple of days.